src/test/java/seedu/address/storage/JsonAdaptedTaskTest.java

```diff
@@ -35,16 +37,18 @@ public void toModelType_validTaskDetails_returnsTask() throws Exception {
     }
 
     @Test
-    public void toModelType_invalidName_throwsIllegalValueException() {
+    public void toModelType_invalidDescription_throwsIllegalValueException() {
         JsonAdaptedTask task =
-                new JsonAdaptedTask(INVALID_DESCRIPTION, VALID_TAGS, false, VALID_NOTE, TEST_DEADLINE);
+                new JsonAdaptedTask(INVALID_DESCRIPTION, VALID_TAGS, false, VALID_NOTE, TEST_DEADLINE
+                        , TEST_PRIORITY);
```

Comment:
hmm is this consistent with style guide haha, is it better to have comma at the end instead of newline?